content/es/docs/tasks/run-application/configure-pdb.md

```diff
@@ -0,0 +1,230 @@
+- Trabajos por lote reiniciables:
+  - Objetivo: El trabajo debe completarse en caso de una interrupción voluntaria.
+    - Posible solución: No cree un PDB. El controlador de Jobs creará un pod de reemplazo.
+
+## Especificando un PodDisruptionBudget
+
+Un `PodDisruptionBudget` tiene tres atributos:
+
+* Un label selector `.spec.selector` para especificar el grupo de
+pods donde aplicar el presupuesto. Este campo es requerido.
+* `.spec.minAvailable` que es una descripción del número de pods del grupo que deben estar disponibles después del desalojo, incluso en ausencia del pod desalojado. `minAvailable` puede ser un número absoluto o un porcentaje.
+* `.spec.maxUnavailable` (disponible en Kubernetes 1.7 y superior) que es una descripción
+del numero de pods del grupo que pueden estar indisponibles despues del desalojo. Puede ser un número absoluto o un porcentaje.
+
+{{< note >}}
+Para las versiones 1.8 y anteriores: al crear un `PodDisruptionBudget`
+utilizando la herramienta de línea de comandos `kubectl`, el campo `minAvailable` es 1 por defecto si no se especifica `minAvailable` ni `maxUnavailable`.
+{{< /note >}}
+
+Puedes especificar únicamente un valor para `maxUnavailable` y `minAvailable` por `PodDisruptionBudget`.
+`maxUnavailable` solo se puede usar para controlar el desalojo de pods
+que tienen un controlador asociado manejándolos. En los ejemplos a continuación, "réplicas deseadas"
+hace referencia al valor 'scale' del controlador que gestiona el grupo de pods seleccionados por el
+`PodDisruptionBudget`.
+
+Ejemplo 1: Con un `minAvailable` de 5, se permiten los desalojos siempre que dejen
+5 o más pods disponibles entre las seleccionadas por el `selector` del PodDisruptionBudget.
+
+Ejemplo 2: Con un `minAvailable` del 30%, se permiten los desalojos mientras que al menos 30% de la cantidad de réplicas se mantengan disponibles.
+
+Ejemplo 3: Con un `maxUnavailable` de 5, se permiten desalojos siempre que haya como máximo 5
+réplicas indisponibles entre el número total de réplicas deseadas.
+
+Ejemplo 4: Con un `maxUnavailable` de 30%, se permiten los desalojos siempre y cuando no más del 30%
+de las réplicas esten indisponibles.
+
+En el uso típico, se usaría un solo presupuesto para una colección de pods administrados por
+un controlador, por ejemplo, los pods en un solo ReplicaSet o StatefulSet.
+
+{{< note >}}
+Un presupuesto de disrupción no garantiza que el número/porcentaje de pods especificado 
+siempre estarán disponibles. Por ejemplo, un nodo que alberga un
+pod del grupo puede fallar cuando el grupo está en el tamaño mínimo
+especificados en el presupuesto, lo que hace que el número de pods disponibles este por debajo del tamaño especificado. El presupuesto solo puede proteger contra
+desalojos voluntarios, pero no todas las causas de indisponibilidad.
+{{< /note >}}
+
+Un `maxUnavailable` de 0% (o 0) o un `minAvailable` de 100% (o igual al
+número de réplicas) puede bloquear los drenajes de los nodos por completo.
```

Comment:
No se si esta oración haga sentido? `puede bloquear los drenajes de los nodos por completo.` ya que es mas una traducción literal de `drain`. Personalmente he usado mas *purgar* que *drenar* recursos de computo.

Reply:
Que tal:
> Un `maxUnavailable` de 0% (o 0) o un `minAvailable` de 100% (o igual al
número de réplicas) puede prevenir que los nodos sean purgados completamente.
Esto está permitido según la semántica de `PodDisruptionBudget`.

No estoy seguro de purgados vs drenados, pero usé purgados en este caso a ver que tal.

Reply:
Mmmm suena raro, pero quiza haga sentido

Reply:
Open to suggestions!

Reply:
Yo apostaría más por drenar, al final no estás limpiando un nodo, estás sacando las cargas de trabajo y moviéndolas a otro nodo.

- AWS utiliza vaciado cuando habla de _Connection draining_:

https://docs.aws.amazon.com/es_es/elasticloadbalancing/latest/classic/config-conn-drain.html

- Azure utiliza drenaje

https://docs.microsoft.com/es-es/azure/application-gateway/application-gateway-faq

Se podría buscar algo como _vaciados_, pero yo creo que el verbo _drenar_ se acerca más a la palabra original y eso ayuda a relacionarlo rápidamente con el término inglés. Además, en inglés tiene el mismo significado y se ha extendido su uso a la terminología de servidores / red.

Reply:
Quizá drenar sea mas afin al argot de Cloud y Containers